media-plugins/kodi-pvr-vdr-vnsi/kodi-pvr-vdr-vnsi-9999.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/kodi-pvr/pvr.vdr.vnsi.git"
```

Comment:
Location just got moved to https://github.com/FernetMenta/pvr.vdr.vnsi/:
https://github.com/kodi-pvr/pvr.vdr.vnsi/commit/36023a3675f4a52c43676951046540f1d274061d

Already rolling on to Kodi 18:
https://github.com/FernetMenta/pvr.vdr.vnsi/commit/4d17aaf4ff4d94f9d355365dc516a6859f685037

2.6.9 has been working fine for me on Kodi 17 beta 6 (and seems OK on beta 7)
https://github.com/FernetMenta/pvr.vdr.vnsi/commit/4b06d83e22ac7a30629dd0edf81c5b985cb0133f